intro/scipy/index.rst

```diff
@@ -79,57 +79,55 @@ Scipy : high-level scientific computing
 
 File input/output: :mod:`scipy.io`
 ----------------------------------
+:mod:`scipy.io` contains functions for loading and saving data in
+several common formats including Matlab, IDL, Matrix Market, and
+Harwell-Boeing.
```

Comment:
Should each section begin with a brief introduction to the subpackage?

Reply:
Seems reasonable.